package.json

```diff
@@ -1,9 +1,14 @@
 {
+  "engines": {
+    "node": "16.6.0"
+  },
   "scripts": {
     "fmt": "eslint . --fix",
     "lint": "tsc --noEmit && eslint .",
     "deploy:commands": "ts-node deploy-commands.ts",
-    "dev": "ts-node index.ts"
+    "dev": "ts-node index.ts",
+    "build": "tsc",
+    "postinstall": "tsc"
```

Comment:
Why do we need both build and postinstall?

Reply:
`build` is for local building, in case you want to check building and running JS on your own machine.
`postinstall` is something that Heroku will run after it does `npm install`.